src/pyOpenMS/pxds/FileHandler.pxd

```diff
@@ -26,11 +26,46 @@ cdef extern from "<OpenMS/FORMAT/FileHandler.h>" namespace "OpenMS":
         FileHandler(FileHandler) nogil except +
 
         bool loadExperiment(String, MSExperiment &) nogil except+
+            # wrap-doc:
+            #   Loads a file into an MSExperiment
+            #   -----
+            #   :param filename: The file name of the file to load.
```

Comment:
remove the dot from the end of the lines to stay consistent.

Reply:
Fixed